### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dreyhiflden**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  .container {
+    height: 300px;
+    width: 250px;
+    background: red;
+  }
+
+  .rect {
+    fill:yellow;
```
Comment (carrot-e) [inline css/style.css]: Please check the code style (here and below).

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    stroke:black;
+    stroke-width:1;
+    opacity:0.5
```
Comment (carrot-e) [inline css/style.css]: Missing semicolon

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    width: 800px;
+    background: green;
+  }
+
+  .image {
+    height: 200px;
+    width: 400px;
+    transform: translate(calc(50% - 200px), 60px);
+  }
+}
```
Comment (carrot-e) [inline css/style.css]: No newline.

**Comment by dreyhiflden** [comment]
All changes set.

**Pull request merged**